src/tests/Microsoft.PowerFx.Interpreter.Tests/PrettyPrintTests.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+
+namespace Microsoft.PowerFx.Interpreter.Tests
+{
+    public class PrettyPrintTests
+    {
+        [Theory]
+        [InlineData("((1 + 2) + 3)", "1 + 2 + 3")]

```

Comment:
>((1 + 2) + 3)", "1 + 2 + 3" [](http://example.com/codeflow?start=21&length=27)

Looking at the tests, is it really doing pretty printing, or only removing unnecessary parentheses? If the latter, we should rename it. If the former, we should have more comprehensive tests.

Reply:
Trying to test if it remove non necessary characters in expression. We already have existing test for multiple cases in public void TestPrettyPrint(string script, string expected) in FormatterTests file. 
The reason I create this function so I can call it from our internal powerfx repo for NL2FX purpose. The original prettyprint function is internal that cannot be used.
Update the name to be less confused.